examples/kmeans/kmeans.py

```diff
@@ -0,0 +1,93 @@
+try:
+    import matplotlib
+    matplotlib.use('Agg')
+    import matplotlib.pyplot as plt
+except ImportError:
+    pass
+
+import numpy as np
+
+import cupy
+
+
+class KMeans(object):
```

Comment:
At first, I wanted to make the interface same to sklearn's one. However, I think there is no need to do that now.